apps/condo/domains/ticket/utils/serverSchema/exportTicketBlanksToPdf.js

```diff
@@ -140,24 +148,24 @@ const getAddressPartsByTicket = ({ ticket, locale }) => {
     }
 }
 
-const convertCommentsToTicketBlank = (comments) => {
+const convertCommentsToTicketBlank = ({ comments, timeZone }) => {
     return comments.map(comment => ({
         ...comment,
-        createdAt: dayjs(comment.createdAt).format('DD.MM.YYYY, HH:mm'),
+        createdAt: formatDate(comment.createdAt, timeZone, 'DD.MM.YYYY, HH:mm'),
     }))
 }
 
-const convertTicketToTicketBlank = ({ ticket, comments, blankOptions, locale }) => {
+const convertTicketToTicketBlank = ({ ticket, comments, blankOptions, locale, timeZone }) => {
     return {
         number: ticket.number,
-        printDate: dayjs().format('DD.MM.YYYY'),
-        createdAt: dayjs(ticket.createdAt).format('DD.MM.YYYY'),
+        printDate: formatDate(undefined, timeZone),
```

Comment:
Not best practice to pass undefined values followed by defined vars....

If some arg is optional, it's better to pass strict `null` value, or use simple object as args `{ timeZone: string, date?: string }`

Reply:
Fixed it